pkg/envtest/helper.go

```diff
@@ -3,10 +3,19 @@ package envtest
 import (
 	"reflect"
 
+	apiextensionsinstall "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+var (
+	crdScheme = runtime.NewScheme()
+)
+
+func init() {
```

Comment:
in general I at least want to discuss why we have an `init` anywhere that this is required. This is for integration tests, and I _strongly believe_ it generally makes sense, but maybe some inline documentation that this is an important external dependency for tests would help for future PRs that affect this code.

I've just been bit enough by init that when encountering it we should:

1) remove it
2) if not remove it, have very strong documentation for it that is well maintained.